server/cmd/main.go

```diff
@@ -273,6 +276,46 @@ func main() {
 	connToInstanceTracker := handlers.ConnectionToStateMachineInstanceTracker{
 		ConnectToInstanceMap: make(map[uuid.UUID]*machines.StateMachine, 0),
 	}
+	hosts, _, err := regManager.GetRegistrants(&v1alpha1.HostFilter{})
+	if err != nil {
+		log.Error(err)
+	}
+	for _, host := range hosts {
```

Comment:
@Jougan-0 The registration of models happens concurrently. How can you guarantee that all models have been registered by the time this function runs? 
Why you are counting after the registration is completed? You can maintain the error count & details, as soon as the registration fails for a particular model then after the registration function exists you can write that data to the file.

Reply:
you are right I should move it to execute in this 
```https://github.com/meshery/meshery/blob/cfd822700dc4dcb7b4cafa9d3af593fee017cf31/server/meshmodel/helper.go#L182```